news/2018-09-18-the-qat-gazette-september-2018.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "The QAT Gazette: September 2018"
+date: 2018-10-18 12:00:00 +0500
+---
+
+Huge changes to Beatmap Nominator applications, updates on improvements to Modding Discussions, and a brand new contributor? What will they come up with next? This month’s issue of the QAT Gazette has all the answers!
```

Comment:
I assumed contributor meant profile badge contributor, should change to avoid confusion

Reply:
Good point, I'll clarify this as "blog contributor"